BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/Onboarding/DigitalInvoiceOnboardingNavigationBarBottomAdapter.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  DigitalInvoiceOnboardingNavigationBarBottomAdapter.swift
+//  
+//
+//  Created by David Vizaknai on 10.02.2023.
+//
+
+import GiniCaptureSDK
+import UIKit
+
+/**
+Protocol for injecting a custom bottom navigation bar on the onboarding screen for digital invoices.
+
+- note: Bottom navigation only.
+*/
+public protocol DigitalInvoiceOnboardingNavigationBarBottomAdapter: InjectedViewAdapter {
+    /**
+     *  Set the callback for the continue button action.
+     *
+     * - Parameter callback: An  action callback, which should be retained and called in get started button action method
+     */
+    func setContinueButtonClickedActionCallback(_ callback: @escaping  () -> Void)
```

Comment:
[Request] There was an inconsistency in the designs in Figma. The button title "Continue" was wrong. It should have been "Get started". I fixed the [designs](https://www.figma.com/file/m4OtCvxWxPMeB0Z5k4Aeow/Gini-Photo-Payment-%2F-iOS?node-id=1094%3A21729&t=fzl3VYsWSuD7JlHL-0) and I need to ask you to rename this method to `setGetStartedButtonClickedActionCallback()` and change the button title to "Get started"/"Los geht's". Sorry 🙈

Reply:
Fixed :D

<img width="988" alt="image" src="https://user-images.githubusercontent.com/19169669/218969943-e95b6cbf-3740-4304-ba40-58ed9d75d28b.png">